apps/web/src/app/(dashboard)/documents/[id]/document-page-view.tsx

```diff
@@ -86,12 +88,15 @@ export const DocumentPageView = async ({ params, team }: DocumentPageViewProps)
     documentMeta.password = securePassword;
   }
 
-  const [recipients, completedFields] = await Promise.all([
+  const [recipients, pendingFields, completedFields] = await Promise.all([
     getRecipientsForDocument({
       documentId,
       teamId: team?.id,
       userId: user.id,
     }),
+    getPendingFieldsForDocument({
+      documentId,
+    }),
     getCompletedFieldsForDocument({
       documentId,
     }),
```

Comment:
At this point I think you should:
- Get rid of `getPendingFieldsForDocument`
- Get rid of `getCompletedFieldsForDocument` (just from this file)
- Use existing `getFieldsForDocument` and modify it to return whatever you need

Then just use filters to get pending vs completed fields.